po/it.po

```diff
@@ -1,297 +1,314 @@
 
-#: pdfarranger/pdfarranger.py:1249
+#: pdfarranger/pdfarranger.py:1468
 msgid ""
 "Cropping does not remove any content\n"
 "from the PDF file, it only hides it."
 msgstr ""
 "Il ritaglio non rimuove alcun contenuto\n"
 "dal file PDF, lo nasconde solo."
 
-#: pdfarranger/pdfarranger.py:1252
+#: pdfarranger/pdfarranger.py:1471
 msgid "Crop Margins"
 msgstr "Taglia margini"
 
-#: pdfarranger/pdfarranger.py:1262
-#, fuzzy, python-format
+#: pdfarranger/pdfarranger.py:1481
+#, python-format
 msgid "% of width"
 msgstr "% in larghezza"
 
-#: pdfarranger/pdfarranger.py:1262
-#, fuzzy, python-format
+#: pdfarranger/pdfarranger.py:1481
+#, python-format
 msgid "% of height"
 msgstr "% in altezza"
 
-#: pdfarranger/pdfarranger.py:1369
+#: pdfarranger/pdfarranger.py:1588
 #, python-format
 msgid ""
 "%s is a tool for rearranging and modifying PDF files. Developed using GTK+ "
 "and Python"
 msgstr ""
-"%s è uno strumento per riordinare e modificare file PDF. Sviluppato usando "
-"GTK+ e Python"
+"%s è uno strumento per riordinare e modificare file PDF. Sviluppato con GTK"
+"+ e Python"
+
+#: pdfarranger/pdfarranger.py:1632
+msgid "Unknown file format"
+msgstr "Formato file sconosciuto"
+
+#: pdfarranger/pdfarranger.py:1644
+msgid "Image files are only supported with img2pdf"
+msgstr "I file di immagine sono supportati solo con img2pdf"
 
-#: pdfarranger/pdfarranger.py:1449
+#: pdfarranger/pdfarranger.py:1656
+msgid "Image format is not supported by img2pdf"
+msgstr "Il formato dell'immagine non è supportato da img2pdf"
+
+#: pdfarranger/pdfarranger.py:1658
+msgid "File is neither pdf nor image"
+msgstr "Il file non è un pdf o una immagine"
+
+#: pdfarranger/pdfarranger.py:1709
 msgid "page"
 msgstr "pagina"
 
 #: data/pdfarranger.ui.h:1
 msgid "Open a file and append it to the current document"
-msgstr "Apri un file e accodalo al documento corrente"
+msgstr "Apri un file e aggiungilo al documento corrente"
 
 #: data/pdfarranger.ui.h:2
 msgid "Save"
 msgstr "Salva"
 
 #: data/pdfarranger.ui.h:3
 msgid "Save As"
-msgstr "Salva Come"
+msgstr "Salva come"
 
 #: data/pdfarranger.ui.h:4
 msgid "Number of Pages"
 msgstr "Numero di Pagine"
 
 #: data/pdfarranger.ui.h:5
 msgid "Main Menu"
-msgstr "Menu Principale"
+msgstr "Menù Principale"
 
 #: data/pdfarranger.ui.h:6
 msgid "Zoom In"
-msgstr "Ingrandisci"
+msgstr "Zoom avanti"
 
 #: data/pdfarranger.ui.h:7
 msgid "Zoom Out"
-msgstr "Rimpicciolisci"
+msgstr "Zoom indietro"
 
 #: data/pdfarranger.ui.h:8
 msgid "Rotate Left"
-msgstr "Ruota a Sinistra"
+msgstr "Ruota a sinistra"
 
 #: data/pdfarranger.ui.h:9
 msgid "Rotate Right"
-msgstr "Ruota a Destra"
+msgstr "Ruota a destra"
 
 #: data/menu.ui.h:1
 msgid "_Import"
 msgstr "_Importa"
 
-#: data/menu.ui.h:2
-msgid "_Save"
-msgstr "_Salva"
-
 #: data/menu.ui.h:3
 msgid "Save _As…"
-msgstr "Salva Come…"
+msgstr "Save come…"
 
 #: data/menu.ui.h:4
 msgid "E_xport Selection…"
-msgstr "E_sporta Selezione…"
+msgstr "Esporta Selezione…"
```

Comment:
Why are you removing the accelerators? They are useful for keyboard navigation imo. Press/hold the Alt key in menus to see them in action. 

This is just the first occurrence, there are more changes like this below.